code/gui/serialbox.cpp

```diff
@@ -51,9 +51,43 @@ SerialBox::SerialBox(
         m_status = SerialStatus::CONNECTED;
     }
     update_status(m_status);
+
+    // Setup sliders
+    ui->up_slider->setTickInterval(Power::POWER_INTERVAL);
```

Comment:
there's a lot of repeated code here

can you make like a `setup_power_slider(QSlider &slider)` function

Reply:
Yeah should be able to.